easybuild/easyconfigs/a/AOCC/AOCC-2.3.0-GCCcore-9.3.0.eb

```diff
@@ -0,0 +1,42 @@
+easyblock = 'Binary'
+
+name = 'AOCC'
+version = '2.3.0'
+
+homepage = 'https://developer.amd.com/amd-aocc/'
+description = "AMD Optimized C/C++ & Fortran compilers (AOCC) based on LLVM 11.0"
+
+# Clang also depends on libstdc++ during runtime, but this dependency is
+# already specified as the toolchain.
+toolchain = {'name': 'GCCcore', 'version': '9.3.0'}
+
+sources = ['aocc-compiler-%(version)s.tar']
```

Comment:
No `source_urls` to enable auto-download?

Reply:
I was not able to get a working link. One needs to first accept the EULA and then can download the software.

Reply:
Actually, there is a direct link:
http://developer.amd.com/wordpress/media/files/aocc-compiler-2.3.0.tar

Reply:
> I was not able to get a working link. One needs to first accept the EULA and then can download the software.

I experienced the same issue when I worked on the SPACK package for AOCC.
I decided to contacted AMD to tell me the 'real' location of the tarball.
They told me that they are going to do it themselves and let me know once it's done to review it.

Reply:
I'm not sure about the legal aspect but EULA should be accepted somehow.
http://developer.amd.com/wordpress/media/files/AOCC_EULA.pdf

Reply:
Thanks! I added the `source_urls`.